src/index.html

```diff
@@ -5,8 +5,325 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="MoyoLogo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#apple" class="list__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="list__item">
+            <a href="#samsung" class="list__link">
+              Samsung
+            </a>
+          </li>
+          <li class="list__item">
+            <a href="#smartphones" class="list__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#pc"
+              class="list__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="list__item">
            <a href="#smartphones" class="list__link">
              Smartphones
            </a>
          </li>

          <li class="list__item">
            <a
              href="#pc"
              class="list__link"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
          </li>
```

Add blank line between multiple sibling blocks of html